tests/test_extruct.py

```diff
@@ -13,26 +13,32 @@ class TestGeneric(unittest.TestCase):
 
     maxDiff = None
 
+    @pytest.mark.xfail
```

Comment:
Why is this marked as xfail?

Reply:
I made some changes in the "tests/samples/songkick/elysianfields.html" file. I added a few corner cases to test my code properly, but this test was failing. I inspected, and the functionality for the open graph protocol is not correct. You may want to test this yourself.

Reply:
I checked. I see that `test1` images were not in the output, and it is because `test1: http://example.org/test1#` is not in the prefix definition. But it is in `xmlns` definition, so it might work as well, so maybe you found an issue here. I visit https://ogp.me/ and examples there uses only the `prefix` but as they are based in RDFa I guess it should also support `xmlns`

But if such is the case, better to treat it as an independent issue to be fixed in a different PR in the future. I propose that you add  `test1: http://example.org/test1#` to the prefixes so that the test passes.

Reply:
I added it to test all my corner cases. I will then need to add another sample file, and I was not sure if that is ideal. If you want me to do that, should I add a new file in the songkick folder itself?

Reply:
Yes, that sounds good. Maybe better to have another example to test your corner cases, and leave this file Ok for the global checks.

In any case with your fix we should be able to remove these lines safely also:
https://github.com/adityas114/extruct/blob/ab3d34e5b5eeed9611e49ee06cc6816d9692988c/tests/test_extruct.py#L21-L29